File: C/Iniciante/1131grenais.c
#include <stdio.h>
int main() {
    int i=0,tipo,n,in,gr,vin=0,vgr=0,em=0;
    do {
        scanf("%d%d",&in,&gr);
        i++;
        if(in==gr){
            em++;
        } else if(in>gr){
            vin++;
        } else if(in<gr){
            vgr++;
        }
        while (1) {
            printf("Novo grenal (1-sim 2-nao)\n");
            scanf("%d", &tipo);
            if(tipo==1 || tipo==2)break;
        }
    }while(tipo!=2);
    printf("%d grenais\n"
           "Inter:%d\n"
           "Gremio:%d\n"
           "Empates:%d\n",i,vin,vgr,em);
    if(vin>vgr){
        printf("Inter venceu mais\n");
    } else if(vin<vgr){
        printf("Gremio venceu mais\n");
    }else if(vin==vgr){
        printf("Nao houve vencedor\n");
    }
    return 0;
}

File: Python/iniciante/1131grenais.py
i = vin = vgr = em = tipo = 0
while tipo != 2:
    ine, gr = [int(i) for i in input().split()]
    i += 1
    if ine == gr:
        em += 1
    elif ine > gr:
        vin += 1
    elif ine < gr:
        vgr += 1
    while True:
        print("Novo grenal (1-sim 2-nao)")
        tipo = int(input())
        if tipo == 1 or tipo == 2:
            break
print(("%d grenais\n"
       "Inter:%d\n"
       "Gremio:%d\n"
       "Empates:%d") % (i, vin, vgr, em))
if vin > vgr:
    print("Inter venceu mais")
elif vin < vgr:
    print("Gremio venceu mais")
elif vin == vgr:
    print("Nao houve vencedor")
